remove-element/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {number[]} nums
+ * @param {number} val
+ * @return {number}
+ */
+let removeElement = function(nums, val) {
```

Comment:
There is same code as in another PR?
https://github.com/piwvisInnowise/inter-ship-algorithm-tasks-danyilk/pull/7/files

Reply:
my bad) 
already fixed